### PR TITLE
Move cfg.DATA dict to be shared accross processes

### PR DESCRIPTION
**Pull request by fmaussion**
This will be needed for later use (e.g. @lilianschuster 's MB models). I'm a bit wary of this change, let's hope this won't have too many impacts.

### CODE REVIEW

File: oggm/cfg.py [hunk 1]
```diff
@@ -857,9 +857,19 @@ def set_manager(manager):
...
+        DATA = new_dict
+
+        new_dict = manager.dict()
+        new_dict.update(LRUHANDLERS)
+        LRUHANDLERS = new_dict
     else:
         DL_VERIFIED = dict(DL_VERIFIED)
         DEM_SOURCE_TABLE = dict(DEM_SOURCE_TABLE)
+        DATA = dict(DATA)
+        LRUHANDLERS = dict(LRUHANDLERS)
```
Comment (fmaussion) [inline oggm/cfg.py:872]: @TimoRoth what was the purpose of this re-cast to a dict? They all are dict per definition....

Comment (fmaussion) [reply oggm/cfg.py:872]: Ah I think I know.

**Comment by fmaussion** [comment]
Yeah OK so I learned something about the mp manager (which I kind of knew already). Objects *stored* in the dict of course can't be modified by the processes, they are meant to be read-only.

Anyways, the LRU handler dict is one of the very useless things I wrote. I think that currently, each process creates its own (which is not bad, but a lot of boilerplate code is useless).

File: oggm/cfg.py (hunk 1 above)
Comment (TimoRoth) [reply oggm/cfg.py]: To get rid of the manager, and only keep the data.

**Pull request merged**